manifests/repo.pp

```diff
@@ -0,0 +1,46 @@
+# Installs yum or apt repository for ClickHouse DBMS.
+#
+# Uses Altinity repository for yum and yandex for apt.
+#
+# @summary installs repository with ClickHouse DBMS
+#
+# @example Simple use
+#   include clickhouse::repo
+#
+#
+class clickhouse::repo inherits clickhouse {
+    case $facts['os']['family'] {
+        'RedHat': {
+            yumrepo { 'altinity_clickhouse':
+                baseurl         => "https://packagecloud.io/altinity/clickhouse/el/${facts['os']['release']['major']}/\$basearch",
```

Comment:
i think we have too much hardcoded stuff in here. we should make nearly everything configurable. for example put the variables in `clickhouse` as suggested above and then use them here so users have the chance to modify some behaviour as they see fit

Reply:
As was discussed, it's the default repositories for CH. If somebody need to add a local repo - repo class isn't included by default